docker-protoc/Dockerfile

```
@@ -1,8 +1,8 @@
 FROM ubuntu:18.04
 
 ENV PROTOC_VERSION=3.11.2
-ENV GO_VERSION=1.13.5
-ENV BUF_VERSION=0.5.0
+ENV GO_VERSION=1.13.7
```

```suggestion
ENV GO_VERSION=1.14
```
might as well bump to 1.14

---

This is an unrelated commit. The PR pulls in all unmerged changes from master.